### PR TITLE
E3FP fix and RDK fingerprint

### PR DESCRIPTION
**Pull request by Arch4ngel21**

### CODE REVIEW

File: benchmarking/benchmark.py [hunk 1]
```diff
@@ -225,15 +226,19 @@ def get_times_e3fp(X: pd.DataFrame):
...
+                    # getting a molecule and the fingerprint
+                    mol, values = conf_gen.generate_conformers(mol)
+                    fps = fprints_from_mol(mol, fprint_params=fprint_params)
 
-                # chose the fingerprint with the lowest energy
-                energies = values[2]
-                fp = fps[np.argmin(energies)].fold(1024)
+                    # chose the fingerprint with the lowest energy
+                    energies = values[2]
+                    fp = fps[np.argmin(energies)].fold(1024)
```
Comment (j-adamczyk) [inline benchmarking/benchmark.py:236]: Do not use constant 1024, this is a constructor parameter

File: benchmarking/benchmark.py [hunk 2]
```diff
@@ -225,15 +226,19 @@ def get_times_e3fp(X: pd.DataFrame):
...
+
+                    X_seq.append(fp.to_vector())
+                except RuntimeError:
+                    X_seq.append(np.full(shape=1024, fill_value=-1))
```
Comment (j-adamczyk) [inline benchmarking/benchmark.py:240]: Do not use constant 1024, this is a constructor parameter

Comment (j-adamczyk) [reply benchmarking/benchmark.py:240]: Also, maybe return all zeros instead of all -1?

File: skfp/fingerprints/rdk.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+from typing import List, Optional, Union
+
+import numpy as np
+import pandas as pd
+import scipy.sparse as spsparse
+
+from skfp.fingerprints.base import FingerprintTransformer
+
+
+class RDKFingerprint(FingerprintTransformer):
```
Comment (j-adamczyk) [inline skfp/fingerprints/rdk.py:10]: I would all this fully `RDKitFingerprint`, since this is the official name

**Review by j-adamczyk — CHANGES_REQUESTED** [review]

File: benchmarking/benchmark.py (hunk 2 above)
Comment (Arch4ngel21) [reply benchmarking/benchmark.py:240]: But isn't this a better way to handle such cases? Vector of -1's is easy to find and, e.g., to be excluded later by the user, while full of zeros is not very apparent and may lead to hidden false information in the data.

File: benchmarking/benchmark.py (hunk 1 above)
Comment (Arch4ngel21) [reply benchmarking/benchmark.py:236]: IMO it's better to leave like this. Either 1. we'd need to define a hyperparameter for folding bits - that means we should pass it as an argument to all fingerprint constructors (that requires it) and that is messing with the current helper function for calculating time for our library, or 2. we'd need to use the parameter's value from E3FP instance, which is not directly accessible from the function above.

File: benchmarking/benchmark.py (hunk 2 above)
Comment (Arch4ngel21) [reply benchmarking/benchmark.py:240]: MAP4, MHFP and E3FP have handling like this (with vector of -1's)

Comment (j-adamczyk) [reply benchmarking/benchmark.py:240]: Ok

File: benchmarking/benchmark.py (hunk 1 above)
Comment (j-adamczyk) [reply benchmarking/benchmark.py:236]: The target dimensionality is the number of bits after folding. I mean, if the default value for E3FP dimensionality is 1024, then it should be 1024 after folding. But for now let's leave this as-is, we can change it later if needed.

**Review by j-adamczyk — APPROVED** [review]

**Pull request merged**